pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.mypy]
```

Comment:
If we add a `pyproject.toml` in the long term we should probably move configuration from https://github.com/pyodide/pyodide/blob/main/setup.cfg here as well.